### PR TITLE
Add Universal Analytics docs.

### PR DESCRIPTION
**Pull request by rdwallis**
Only the commits after fix apostrophe matter.  Not sure how to clean up this PR to get rid of the commits that are already in master.

Couldn't immediately work out how to create the menu items for Universal Analytics.  Can someone else do that for me?  We also need a Universal Analytics logo.

### CODE REVIEW

**Comment by spg** [comment]
@rdwallis I can do that for you, I'll add an issue to Jira. Do you have access to our Jira?

**Comment by rdwallis** [comment]
@spg thanks.  No I don't have access to Jira.

**Comment by spg** [comment]
@christiangoudreau is it possible to give @rdwallis access to our Jira, so he can create issues if he wants?

**Comment by meriouma** [comment]
@christiangoudreau Ping :)

**Comment by christiangoudreau** [comment]
Totally didn't see this one :D

**Comment by christiangoudreau** [comment]
LGTM

File: site/pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <url>http://www.gwtproject.org</url>
 
     <properties>
-        <build-tools.version>1.0-SNAPSHOT</build-tools.version>
+        <build-tools.version>1.0</build-tools.version>
```
Comment (Chris-V) [inline site/pom.xml]: Think it's 1.1 now

**Comment by Chris-V** [comment]
LGTM

**Comment by rdwallis** [comment]
@spg can you take ownership of this PR and do the merge once you think it's ready?  All the text is there it just needs the logo and menu items.

**Pull request merged**